src/storage/exec/FilterNode.h

```diff
@@ -22,7 +22,7 @@ data, but not both.
 
 As for GetNeighbors, it will have filter that involves both tag and edge
 expression. In that case, FilterNode has a upstream of HashJoinNode, which will
-keep popping out edge data. All tage data has been put into ExpressionContext
+keep popping out edge data. All tag data has been put into ExpressionContext
```

Comment:
possibly `tagged` (but `tag` was closer, and the initial word corrections were picked by Google Sheets) - I don't have enough confidence to override that suggestion.

Reply:
`tag` is just ok. It is an existential concept similar to `edge data`.